### ISSUE TITLE
Default queue with unique jobs.

### ISSUE THREAD

**Issue by mindreader** [sorentwo/oban#71]
It seems like no matter what I do, I cannot seem to use the "default" queue while also using uniqueness.

```
config :simple, Oban,
  repo: Simple.DB.Postgres,                                                                  
  queues: [default: 10, foo: 2],                                                        
  verbose: false,
  prune: {:maxage, 60 * 60 * 24}
```

The job
```
  use Oban.Worker,
    max_attempts: 4,
    unique: [period: 3600, states: [:available, :scheduled, :executing]]

def foo do
    %{bar: 123}
    |> IO.inspect()
    |> __MODULE__.new()
    |> Oban.insert()
end

```
** (ArgumentError) nil given for :queue. Comparison with nil is forbidden as it is unsafe. Instead write a query with is_nil/1, for example: is_nil(s.queue)
    (ecto) lib/ecto/query/builder/filter.ex:128: Ecto.Query.Builder.Filter.kw!/7
    (ecto) lib/ecto/query/builder/filter.ex:121: Ecto.Query.Builder.Filter.kw!/3
    (ecto) lib/ecto/query/builder/filter.ex:103: Ecto.Query.Builder.Filter.filter!/6
    (ecto) lib/ecto/query/builder/filter.ex:115: Ecto.Query.Builder.Filter.filter!/7
    (oban) lib/oban/query.ex:202: Oban.Query.get_unique_job/3
    (oban) lib/oban/query.ex:37: Oban.Query.fetch_or_insert_job/2
```

**Comment by sorentwo** [comment]
I found the same thing earlier this week. This is fixed in v0.9.0, which. Was released earlier today.

**Comment by mindreader** [comment]
I'm sorry, I should have looked!  I thought I had the most up to date version!  Thanks!
